src/test/codegen/issue-34947-pow-i32.rs

```diff
@@ -6,8 +6,8 @@
 #[no_mangle]
 pub fn issue_34947(x: i32) -> i32 {
     // CHECK: mul
-    // CHECK-NEXT: mul
-    // CHECK-NEXT: mul
-    // CHECK-NEXT: ret
```

Comment:
This is a regression test to ensure that `pow(<constant>)` would unroll the loop properly. This change regresses that and the test adjusted to ignore its original intent.

Reply:
See https://github.com/rust-lang/rust/issues/34947

Reply:
I don't think `pow(<constant>)` regresses in this case. The codegen simply inserts a bunch of `@llvm.sideeffect` in between `mul` so we can't do `CHECK-NEXT`. The resulting code should be no different.

Reply:
Can we then `CHECK-NOT` for branches between the multiply instructions?

Reply:
Sure.